docs/specs/add-kubernetes-support.md

```diff
@@ -28,20 +28,62 @@ Start by creating a new pkg for cluster objects, ```pkg/cluster/...```. From
 there, the cluster pkg will hold ```pkg/cluster/cluster.go```,
 ```pkg/cluster/kubernetes.go```, and ```pkg/cluster/openshift.go```.
 
-The file ```pkg/cluster/cluster.go```  will provide an object that will abstract
+The file ```pkg/cluster/cluster.go``` will provide an object that will abstract
 cluster logic.
+```diff
++type COE interface {
++     CreateSandbox()
++     Run(APBActionProvision APBAction, req Context)
++     DestroySandbox()
++}
+
++struct Cluster {
```

Comment:
@eriknelson that's true, Docker would not be a cluster.  Something like Delegate wfm or maybe Runtime?